pom.xml

```diff
@@ -202,7 +182,7 @@
         <dependency>
```

Comment:
Check whether you can remove oauth2 dependency as well.

Reply:
No, Build is failing because of the direct dependency in one of the class.